pkg/domain/infra/abi/images.go

```diff
@@ -549,3 +549,8 @@ func (ir *ImageEngine) Remove(ctx context.Context, images []string, opts entitie
 
 	return
 }
+
+// Shutdown Libpod engine
```

Comment:
One of these (containers/images shutdown) is guaranteed to error, duplicate shutdown will never work.